### PR TITLE
Update troubleshooting button

### PR DESCRIPTION
**Pull request by salarson**

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: OneDrive/change-user-storage.md [hunk 1]
```diff
@@ -122,3 +122,13 @@ When you need cloud storage for individual users beyond the initial 5 TB, additi
 > To change the storage space for multiple users, use PowerShell to [Display a list of OneDrive accounts by using PowerShell](list-onedrive-urls.md) and use [Set-SPOSite](/powershell/module/sharepoint-online/set-sposite?preserve-view=true&view=sharepoint-ps&preserve-view=true) to make the change.
 > 
 > To disable OneDrive creation for specific users, see [Manage user profiles in the SharePoint admin center](/sharepoint/manage-user-profiles).
+
+## Automated Troubleshooting for issues changing OneDrive Storage
```
Comment (JohanFreelancer9) [inline OneDrive/change-user-storage.md:126]:
```suggestion
## Automated troubleshooting for issues when changing OneDrive storage
```

File: OneDrive/change-user-storage.md [hunk 2]
```diff
@@ -122,3 +122,13 @@ When you need cloud storage for individual users beyond the initial 5 TB, additi
...
+
+If you're an administrator, and you're having trouble changing a users OneDrive Storage, select Run Tests below, which will populate a diagnostic in the Microsoft 365 Admin Center. These tests will check your configuration and quickly recommend next steps to resolve the issue.
```
Comment (JohanFreelancer9) [inline OneDrive/change-user-storage.md:128]:
```suggestion
If you are an administrator, and you're having trouble changing a user's OneDrive storage, select the following Run Tests: OneDrive Storage Quota, which will populate a diagnostic in the Microsoft 365 admin center. These tests will check your configuration and quickly recommend the next steps to resolve the issue. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @salarson, kindly commit the suggested change if you agree with it. Many thanks.

**Pull request merged**